cloudhub/modules/ROOT/pages/app-migration.adoc

```diff
@@ -0,0 +1,142 @@
+= CloudHub to CloudHub 2.0 Application Migration
+
+After your VPC upgrade is complete, you can start migrating all associated applications. You can do so by creating an equivalent CloudHub 2.0 application for every CloudHub application, and linking them to have weighted DNS records created for both the CloudHub and the CloudHub 2.0 URLs. 
+
+
+The current CloudHub application continues to run without any disruption until the upgrade is complete. After the upgrade is complete, you can continue to use the same CloudHub URL for your new CloudHub 2.0 application.
+
+Some configuration settings of the CloudHub application aren't automatically mapped to the new CloudHub 2.0 application. Go to your new CloudHub 2.0 application settings to finish the configuration.
+
+[NOTE]
+During the application upgrade process, no out-of-the-box entitlements are provided. Work with your account representative to remain compliant with entitlements and finish the upgrade on time.
+
+== Upgrade your CH application to CH2 via Anypoint Platform
+
+. From *Anypoint Platform*, select *Runtime Manager* > *Applications*.
+. Select the CloudHub application you want to upgrade and navigate to *Settings*. 
+. If your application is eligible for the upgrade, the upgrade tooling is available on the *Settings* page: click *Upgrade to CloudHub 2.0*, then click *Start Upgrade*.
++
+image::upgrade-application-window.png[CloudHub to CLoudHub 2.0 application upgrade window.]
++
+The *Upgrade Application to CloudHub 2.0* page opens.
++
+. The *Application Name* field shows the inherited CloudHub application name by default. If you want to use a different name for your CloudHub 2.0 application, change it by editing the field.
```

Comment:
```suggestion
The *Application Name* field shows the inherited CloudHub application name by default. To use a different name for your CloudHub 2.0 application, change it by editing the field.
```

Reply:
I made this not a step because it doesn't start with an imperative. And it's optional.